docs/development/extensions-core/kinesis-ingestion.md

```diff
@@ -46,7 +41,10 @@ For example:
 curl -X POST -H 'Content-Type: application/json' -d @supervisor-spec.json http://localhost:8090/druid/indexer/v1/supervisor
```

Comment:
imo it would be better to use a minimal inline supervisor spec in the payload rather than to link to a file. Using tabbed cURL & HTTP examples. See the sample request here: https://github.com/apache/druid/pull/14492/